### ISSUE TITLE
Problems Firejailing Discord on Linux Mint

### ISSUE THREAD

**Issue by DangerousDays** [netblue30/firejail#4341]
When  type the command:

`firejail discord`

I get the following text:

```
Reading profile /etc/firejail/discord.profile
Reading profile /etc/firejail/discord-common.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-passwdmgr.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/whitelist-common.inc
Reading profile /etc/firejail/whitelist-var-common.inc
Warning: networking feature is disabled in Firejail configuration file
Parent pid 170933, child pid 170934
Warning: skipping crypto-policies for private /etc
Warning: skipping password for private /etc
Private /etc installed in 25.90 ms
Warning: skipping discord for private /opt
Private /opt installed in 0.13 ms
13 programs installed in 14.88 ms
Warning: cleaning all supplementary groups
Warning: cleaning all supplementary groups
Child process initialized in 93.38 ms

Parent is shutting down, bye...
```

The discord app was installed using the official .deb. Other programs like firefox and signal work just fine when I type `firejail [program]` into the terminal.

**Comment by SkewedZeppelin** [comment]
What version of firejail are you running?
Can you paste the output of `firejail --version`?

**Comment by DangerousDays** [comment]
I am running 0.9.62

**Comment by rusty-snake** [comment]
There were a lot fixes since 0.9.62. Can you reproduce this with firejail 0.9.64.4/0.9.65?

Is anything in the syslog?

**Comment by rusty-snake** [comment]
Duplicate of #3634, #3328, #3247

**Comment by DangerousDays** [comment]
Thank you. The bug was fixed in 0.9.64.
